contrib/examples/generate-blueprint-docs-simple/README.md

```diff
@@ -0,0 +1,32 @@
+# generate-blueprint-docs: Simple Example
+
+### Overview
+
+This example shows how the [generate-blueprint-docs] function works.
+
+Running `generate-blueprint-docs` function on the example package will:
+
+1. Generate a `GENERATED.md` file with the blueprint readme.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/contrib/examples/generate-blueprint-docs-simple
+```
+
+### Function invocation
+
+Invoke the function with the following command:
+
+```shell
+kpt fn eval -i gcr.io/kpt-fn-contrib/blueprint-docs:unstable --include-meta-resources \
+--mount type=bind,src="$(pwd)",dst="/tmp",rw=true -- readme-path=/tmp/GENERATED.md
+```
+
+### Expected result
+
+1. File `GENERATED.md` will be created with the generated readme.
+
+[generate-blueprint-docs] https://catalog.kpt.dev/generate-blueprint-docs/v0.1/?id=definitions
```

Comment:
> ?id=definitions

This is not needed, right?

Reply:
The markdown was off but I do use it here https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/caf0fd64b5153d094c68acbc2a71288cc97c55f8/contrib/examples/generate-blueprint-docs-simple/README.md?plain=1#L5

Reply:
I meant why we have a query parameter `id=definitions` in the URL. AFAIK our fn catalog site doesn't support this query parameter.

Reply:
@mengqiy oops removed. I copied from [another example](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/471a6a2dccd0f3e0da15fd55ccefe6a7e22a3fcb/contrib/examples/annotate-apply-time-mutations-simple/README.md?plain=1#L32)